### DISCUSSION TITLE
How can i implement such pagination on react?

### DISCUSSION THREAD

**Discussion by sl1m41kc** [nolimits4web/swiper#7686] [Q&A] [answered]
Hello, my task is to implement this kind of pagination:
![image](https://github.com/user-attachments/assets/835339a5-56ee-4a5a-bd2a-4ba365dd0f1e)
But the documentation is too complex to understand...

P.S. Sorry for my English, I used the deepl translator :)

**Comment by azharuddin-mulla-amla** [comment, marked as answer]
@sl1m41kc oh you can customize your pagination bullet by using their existing css class name.

/* Pagination Bullet Container */
.swiper-horizontal > .swiper-pagination-bullets,
.swiper-pagination-bullets.swiper-pagination-horizontal {
  bottom: -25px;
}

/* Pagination Bullets */
.swiper-horizontal > .swiper-pagination-bullets .swiper-pagination-bullet,
.swiper-pagination-horizontal.swiper-pagination-bullets
  .swiper-pagination-bullet {
  background-color: red;
  /* custom css  */
}

/* Pagination Bullet Active */
.swiper-pagination-bullet-active {
  background-color: rgba(0, 0, 0, 0.9);
}
